File: Easy/19.SecondGreatLow/solve.rb
require "set"

def SecondGreatLow(arr)
	sortSet = SortedSet.new(arr).to_a

	if sortSet.length == 1
		return sortSet[0].to_s + " " + sortSet[0].to_s
	elsif sortSet.length == 2
		return sortSet[0].to_s + " " + sortSet[1].to_s
	elsif sortSet.length == 3
		return sortSet[1].to_s + " " + sortSet[1].to_s
	else
		return sortSet[1].to_s + " " + sortSet[sortSet.length-2].to_s
	end

end

   
# keep this function call here    
puts SecondGreatLow(STDIN.gets)

File: Easy/33.SuperIncreasing/solve.py
def Superincreasing(arr):
	for x in range(0, len(arr)-1):
		s = sum(arr[0:x+1])
		if arr[x+1] <= s:
			return "false"
	return "true"
    
# keep this function call here  
print Superincreasing(raw_input())